async function main(): Promise<void> {
    console.log("Hello World!")
}

main()
    .then(() => console.log('Finished.'))
    .catch(console.error)
